docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,320 @@
+# manager. IMPORTANT: The local MSP ID of an orderer needs to match the MSP
+# ID of one of the organizations defined in the orderer system channel's
+# /Channel/Orderer configuration. The sample organization defined in the
+# sample configuration provided has an MSP ID of "SampleOrg".
+LocalMSPID: SampleOrg
+```
+
+* **`LocalMSPID`**: (default value should be overridden) The MSP ID must match the orderer organization MSP ID that exists in the configuration of the system channel. This means the MSP ID must have been listed in the `configtx.yaml` used to create the genesis block of the system channel (or have been added later to the list of system channel administrators).
+
+## General.BCCSP.*
+
+```
+# Default specifies the preferred blockchain crypto service provider
+        # to use. If the preferred provider is not available, the software
+        # based provider ("SW") will be used.
+        # Valid providers are:
+        #  - SW: a software based crypto provider
+        #  - PKCS11: a CA hardware security module crypto provider.
+        Default: SW
+
+        # SW configures the software based blockchain crypto provider.
+        SW:
+            # TODO: The default Hash and Security level needs refactoring to be
+            # fully configurable. Changing these defaults requires coordination
+            # SHA2 is hardcoded in several places, not only BCCSP
+            Hash: SHA2
+            Security: 256
+            # Location of key store. If this is unset, a location will be
+            # chosen using: 'LocalMSPDir'/keystore
+            FileKeyStore:
+                KeyStore:
+```
+
+(Optional) This section is used to configure the Blockchain crypto provider.
+
+* **`BCCSP.Default:`** If you plan to use a Hardware Security Module (HSM), then this must be set to `PKCS11`.
+
+```
+# Settings for the PKCS#11 crypto provider (i.e. when DEFAULT: PKCS11)
+        PKCS11:
+            # Location of the PKCS11 module library
+            Library:
+            # Token Label
+            Label:
+            # User PIN
+            Pin:
+            Hash:
+            Security:
+            FileKeyStore:
+                KeyStore:
+```
+
+* **`BCCSP.PKCS11.*:`** Provide this set of parameters according to your HSM configuration. Refer to this [example]((../hsm.html) of an HSM configuration for more information.
+
+## FileLedger.Location
+
+```
+# Location: The directory to store the blocks in.
+Location: /var/hyperledger/production/orderer
+```
+
+* **`Location`**: (default value should be overridden, especially if more than one node will be created by this organization) Every channel on which the node is a consenter will have its own subdirectory at this location. The user running the orderer needs to own and have write access to this directory. **The best practice is to store this data in persistent storage**. This prevents the ledger from being lost if your orderer containers are destroyed for some reason.
```

Comment:
I'm a little confused why having more than one node means this location should be overridden? I guess you could put the host name into the path in the container, but you could also just name your volume properly.

My guess is, you are referring to running two non-containerized binaries on the same host.  But it's odd here that we assume this, while below for the operations endpoint we talk about "in the unlikely event where two ordering nodes are running on the same node"

Reply:
Ok changed this to "(default value should be overridden in the unlikely event where two ordering nodes are running on the same node)"